add-on/panels/js/panel.jsx

```diff
@@ -765,8 +768,10 @@ loop.panel = (function(_, mozL10n) {
                     room={room} />
                 );
               }, this)
-            }</div>
+            }
+            </div>
           }
+          {!(this.state.rooms.length > 5) ? null : <div className="room-list-blur" />}
```

Comment:
Please at least use <= and get rid of the negation, or, even better, ditch the ternary operator here.